dvc/scm/git/__init__.py

```diff
@@ -260,15 +260,12 @@ def get_tree(self, rev):
 
     def _get_diff_trees(self, a_ref, b_ref):
         """Private method for getting the trees and commit hashes of 2 git
-        references.
-        Requires `gitdb` module (from gitpython package).
+        references. Requires `gitdb` module (from gitpython package).
 
-        Args:
-            a_ref(str) - git reference
-            b_ref(str) - second git reference. If None, uses HEAD
+        Args: a_ref(str) - git reference b_ref(str) - second git reference. If
```

Comment:
Not sure why did you reformat these?

Reply:
Same comment, as @efiop ;
We don't have a convention, but we try to follow Google's one: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

It happened naturally, not anything special about it. I dislike the Java (epytext) one, and reST is cool but already started with Google, maybe we can change to https://github.com/dadadel/pyment but it is not worth the effort, this docstrings are for developers, they don't escalate to the user, and we are not publishing an API, so for now, let's stay with the Google one (I really like that they have a section for examples, useful when the code intention is not that clear)

Reply:
I didn't do this on purpose, thanks for noticing! I think I over-cliecked `Alt+Q` in VSCode ([Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)...

Thanks for the Google docstring style ref Ramón, will keep it handy.

These have been fixed now.

Reply:
:+1: